app/components/Navigation/index.ts

```diff
@@ -43,6 +43,11 @@ export class Navigation {
         transformOrigin: '50% 50%',
       })
 
+      GSAP.to('.topnav__link__icon--line2', {
+        duration: 0.5,
+        autoAlpha: 0,
```

Comment:
test thoroughly to make this appears when it's meant to, sometimes the second line disappears and doesn't come back